crates/client/devnet/src/classes.rs

```diff
@@ -24,11 +26,23 @@ pub enum InitiallyDeclaredClass {
 }
 
 impl InitiallyDeclaredClass {
```

Comment:
just out of curiosity, why aren’t the classes definition stored compressed directly?

Reply:
I didn't understand the question sorry, the compressed version is stored now right?

Reply:
it's just that we produce uncompressed class definitions with scarb

Reply:
Oh ok, I am not really sure but if Scarb has that option then this could be an optimisation, yes.